src/components/info-tile/examples/info-tile-notification.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, h } from '@stencil/core';
+/**
+ * Displaying a notification badge
+ *
+ * The component can display a notification badge, which could either be
+ * a `number` or a `string`. Read more about how the badge truncates
+ * or abbreviates the provided label [here](#/component/limel-badge/).
+ *
+ */
+
+@Component({
+    tag: 'limel-example-info-tile-notification',
```

Comment:
Regardless of whether we call the badge "badge" or "notification badge", I think the word "notification" _alone_, without "badge", definitely means something else.

```suggestion
    tag: 'limel-example-info-tile-badge',
```

Reply:
Don't forget to change the filename of this file and the scss file!